tests/accessibility/index.test.js

```diff
@@ -0,0 +1,361 @@
+// @ts-check
+
+// The OpenTelemetry instrumentation for Express breaks our ability to inspect
+// the Express routes. We need to disable it before loading the server.
+const { disableInstrumentations } = require('@prairielearn/opentelemetry');
+disableInstrumentations();
+
+const path = require('path');
+const { test } = require('mocha');
+const axe = require('axe-core');
+const jsdom = require('jsdom');
+const fetch = require('node-fetch').default;
+const { A11yError } = require('@sa11y/format');
+const util = require('util');
+const expressListEndpoints = require('express-list-endpoints');
+const sqldb = require('@prairielearn/postgres');
+
+const server = require('../../server');
+const news_items = require('../../news_items');
+const config = require('../../lib/config');
+const helperServer = require('../helperServer');
+
+const SITE_URL = 'http://localhost:' + config.serverPort;
+const EXAMPLE_COURSE_DIR = path.resolve(__dirname, '..', '..', 'exampleCourse');
+
+/**
+ * Several pages have very large DOMs or attributes that AXE runs very slow on.
+ * None of these elements have accessibility components, so we'll special-case these
+ * and remove them before running AXE.
```

Comment:
I don't love these special cases, but they were the only way I could find to avoid adding another minute to our test execution time.

Reply:
Ya, this is annoying. It would be nice to deep dive on why it takes so long and come up with a better solution but I think its fair to push that off until later.